microprofile/server/src/main/java/io/helidon/microprofile/server/JaxRsService.java

```diff
@@ -357,7 +334,7 @@ public OutputStream writeResponseStatusAndHeaders(long contentLengthParam,
             if (contentLength > 0) {
                 res.header(HeaderValues.create(HeaderNames.CONTENT_LENGTH, String.valueOf(contentLength)));
             }
```

Comment:
Explication: There is one SSE test were the client closes the connection and the server was not aware of it, because it was sending events to client but the were never flushed.